readme/getting-started/feature-definition.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Feature Definition
+---
+
+# What is a Feature?
+
+The set of features that compose your application must be defined before they can be used in assignment.  We think of a "feature" of an application as a way to decompose its overall function into discrete, coherent units.  Our formal, working definition is as follows:
+
+> A feature is an internal component of the system that performs a well defined and relatively isolated set of responsibilities.
+
+It's also helpful to understand what articulating the features of a system _does_:
+
+  - Allows useful conversations about the operation of a given function of the system (e.g., authentication, payment processing)
+  - Allows folks without deep, firsthand knowledge of the details of a given set of source files, to understand the value that they provide as a whole
+  - Allows for extension, refactoring, or other planning activities to consider specific, but holistic units of the system without delving into individual files
+  - In systems where files are organized by type (e.g., models, controllers, views, service classes), allows for source files to be grouped by function without clumsy in-code namespacing
+
+**Here are a few examples of poorly-constructed features**
+
+| Feature | Reason|
+Program Management | Too big
```

Comment:
We should probably avoid domain specific terminology. I'm struggling to think of a feature that would be "too big" in all contexts. We could use something like "All The Things" Feature here as a meta way of describing features that encompass too much responsibility within the system. Thoughts?